.github/workflows/scripts/validate-form.py

```diff
@@ -155,7 +165,132 @@ def validate_form(issue_content_file: str) -> None:
+                            f"required data for CDP event ingestion wasn't found. "
+                            f"A "
+                            f"[cdp-scrapers]"
+                            f"(https://github.com/{COUNCIL_DATA_PROJECT}/cdp-scrapers) "
+                            f"maintainer should look into this issue however it is "
+                            f"likely that you (@{form_values[TARGET_MAINTAINER]}) will "
+                            f"need to write a custom scraper.\n"
+                            f"<summary>Retrieved Data</summary>\n"
+                            f"<details>\n\n"  # Extra new line for proper rendering
+                            f"```json\n"
+                            f"{event_as_json_str}\n"
+                            f"```"
+                            f"</details>"
+                        )
+                        break
+
+        # Catch no video path available
+        # User will need to write a custom Legistar scraper
+        except NotImplementedError:
+            legistar_response = (
+                f":warning: Your municipality uses Legistar but is missing the video "
+                f"URLs for event recordings. We recommended writing a custom Legistar "
+                f"Scraper that inherits from our own to resolve the issue. "
+                f"Please the "
+                f"[cdp-scrapers]"
+                f"(https://github.com/{COUNCIL_DATA_PROJECT}/cdp-scrapers) repository "
+                f"for more details. And please refer to the "
+                f"[SeattleScraper]"
+                f"(https://github.com/{COUNCIL_DATA_PROJECT}/cdp-scrapers"
+                f"/blob/main/cdp_scrapers/instances/seattle.py) for an example of a "
+                f"scraper that inherits from our base to resolve this issue."
+            )
+
+        except Exception:
+            legistar_response = (
+                f":x: Something went wrong during Legistar client data validation. "
+                f"A [cdp-scrapers]"
+                f"(https://github.com/{COUNCIL_DATA_PROJECT}/cdp-scrapers) maintainer "
+                f"should look into the logs for this bug. Sorry about this!"
+            )
+
+    # Handle bad / mis-parametrized legistar info
+    if legistar_response is None:
+        if (
+            form_values[LEGISTAR_CLIENT_ID] is not None
+            and form_values[LEGISTAR_CLIENT_TIMEZONE] is None
+        ):
+            legistar_response = (
+                ":x: You provided a Legistar Client Id but no Timezone. "
+                "**Timezone is required** for Legistar scraping."
+            )
+        else:
+            legistar_response = (
+                ":thought_balloon: **You didn't provided Legistar Client "
+                "information**, please note that you will be required to write "
+                "an entirely custom event scraper after your instance is deployed."
```

Comment:
Would be nice to add something like "Here are the steps for creating a custom event scraper." + link to custom scraper docs.

Also my understanding was that we would require a custom scraper from them to move on to bot 2 but the wording above seems to imply that we will create their repo and deploy their instance and they can add their custom scraper afterwards. Is that correct?

Reply:
1. Totally agree, we need to get the links to the custom scraper docs in here but waiting on those to be merged.

2. Yea I thought about it. It would be really hard for them to hand us a custom scraper.... I can't think of an easy method to do it. So I figured it may just be best to deploy and give them write access to that repo. It's a little clunky but still better than nothing imo. The reason I think it's difficult is because they may have custom dependencies, multiple files, etc. Like it's not just a single file thing, their scraper may be _large_ and heavily custom.

Reply:
I see your comment about adding their scraper to the cdp-scrapers repo... hmmm we could do that would just take some extra work. Let me work on some stuff.